rest_framework_swagger/docgenerator.py

```diff
@@ -365,7 +369,7 @@ def _get_serializer_fields(self, serializer):
                     f['type'] = 'array'
                     if field_serializer:
                         f['items'] = {'$ref': field_serializer}
-                    elif data_type in BaseMethodIntrospector.PRIMITIVES:
```

Comment:
Removing this conditional lets Swagger UI output `array(field)`. If the conditional was left in place, we would just see `array`. The conditional was preventing anything that's not in `BaseMethodIntrospector.PRIMITIVES` from being output. I could've added a `field` key to `PRIMITIVES`, but I didn't think it qualifies as a primitive.

For reference, `PRIMITIVES` looks like this:
```
    PRIMITIVES = {
        'integer': ['int32', 'int64'],
        'number': ['float', 'double'],
        'string': ['string', 'byte', 'date', 'date-time'],
        'boolean': ['boolean'],
    }
```